lib/ip_record.js

```diff
@@ -236,8 +234,8 @@ module.exports = function (limits, now) {
     }
 
     // Increment sms request count and throttle if needed
-    if (actions.isSmsSendingAction(action) && phoneNumber) {
-      this.addSmsRequest({ phoneNumber: phoneNumber })
```

Comment:
The `phoneNumber` argument appears to be unused after removing these, can we safely remote it or does it need to be kept for consistency with other records?